user/urls.py

```diff
@@ -7,7 +7,7 @@
 
 from user.views import CreateUserView, ManageUserView
 
-app_name = "user"
+app_name = "users"

```

Comment:
`app_name `must be the same as `namespace`. Or change here on `user`, or change `namespace` on `users`